doc/unions.adoc

```diff
@@ -182,6 +182,27 @@ function dealWithTree = |elt| -> match {
 }
 ----
 
+=== Default values
+
+Union values members can have default values, in the same way as <<struct-default-values,`struct`>>.
+For instance:
+
+[source,golo]
+----
+union List = {
```

Comment:
My understanding is that I can write something like that:
```golo
union SomeThing = {
  Yum = { value="🍊" }
  Yuck = { value="🍋" }
}
```
and then I hope(expect?) to get:
```golo
Something.Yum("🍆") is union SomeThing.Yum{value=🍆}
Something.Yum() is union SomeThing.Yum{value=🍊}
```

but the call of `Something.Yum()`
generates error:
```
Exception in thread "main" java.lang.invoke.WrongMethodTypeException: cannot convert MethodHandle(Object)SomeThing to ()Object
	at java.lang.invoke.MethodHandle.asTypeUncached(MethodHandle.java:775)
	at java.lang.invoke.MethodHandle.asType(MethodHandle.java:761)
	at org.eclipse.golo.runtime.FunctionCallSupport.fallback(FunctionCallSupport.java:164)
	at main.main(main.golo:33)
	at org.eclipse.golo.cli.command.spi.CliCommand.callRun(CliCommand.java:40)
	at org.eclipse.golo.cli.command.ShebangCommand.execute(ShebangCommand.java:46)
	at org.eclipse.golo.cli.Main.main(Main.java:62)
```

I probably missed something